src/interfaces/store-interfaces/state-interfaces/flags-company-info-interface.ts

```diff
@@ -8,6 +8,7 @@ export interface FlagsCompanyInfoIF {
   isValidCompanyName: boolean
   isValidBusinessType: boolean
   isValidNameTranslation: boolean
+  isValidStartDate: boolean
```

Comment:
Order is important here!

Because isValidStartDate was not in the same order here as in [ComponentsCompanyInfo](https://github.com/bcgov/business-edit-ui/blob/main/src/enums/componentsCompanyInfo.ts), validateAndScroll() [did not identify this component](https://github.com/bcgov/business-edit-ui/blob/34a8a699b5d1fc2242b4a0156cd3f096eb6726fd/src/mixins/common-mixin.ts#L43) and thus it returned True, which let the user go to the Review page instead of staying on the Business page.

Reply:
That is good to know!